connectors/sources/tests/fixtures/mysql/fixture.py

```diff
@@ -22,44 +38,88 @@ def random_text(k=1024 * 20):
 BIG_TEXT = random_text()
 
 
-def inject_lines(table, cursor, start, lines):
-    raws = []
-    for raw_id in range(lines):
-        raw_id += start
-        raws.append((f"user_{raw_id}", raw_id, BIG_TEXT))
-    sql_query = (
-        f"INSERT INTO customers_{table}"
-        + "(name, age, description) VALUES (%s, %s, %s)"
+class Customer:
+    def __init__(self, id_):
+        self.id_ = id_
+        self.name = fake.name()
+        self.description = BIG_TEXT
+
+    def to_row(self):
+        return self.id_, self.name, self.description
```

Comment:
Honestly the class feels rudimentary - I did not look too deep into it, but it looks like you just replace it with a function similar to:

```
def customer_row(id):
    return id, fake.name(), BIG_TEXT
```

Reply:
Fair point, thought about externalising a more complex `Customer` class, which can be reused in different tests. But for this PR it's probably better to just us a one-liner function.

Reply:
TBH I think that functional tests should be as simple as possible. If they get complex, it's a good question - do we really want that complexity?